sdks/java/io/hadoop-file-system/src/test/java/org/apache/beam/sdk/io/hdfs/HadoopFileSystemOptionsTest.java

```diff
@@ -159,6 +239,43 @@ public void testDefaultSetYarnConfDirAndHadoopConfDirNotSameConfiguration() thro
     assertThat(configurationList.get(1 - hadoopConfIndex).get("propertyD"), Matchers.equalTo("D"));
   }
 
+  @Test
```

Comment:
This last test is theoretically correct, but doesn't add much value (there can only be one Configuration today).  I'd get rid of it.

Reply:
Ok, I will remove the test.